upgrading/upgrade-operator.adoc

```diff
@@ -11,14 +11,47 @@ Upgrades through the {rh-rhacs-first} Operator are performed automatically or ma
 
 If you installed {product-title-short} using the Operator and selected *Automatic* in the *Update approval* field, {product-title-short} is automatically updated when a new software version is released. If you selected *Manual*, you must approve subsequent Operator updates by using Operator Lifecycle Manager (OLM). For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update].
 
-To roll back an Operator upgrade, you must perform the steps described in one of the following sections. You can roll back an Operator upgrade by using the CLI or the {ocp} web console.
+[IMPORTANT]
+====
+* Because of the database related changes introduced in {product-title-short} 4.0, even if you have selected *Automatic* in the *Update approval* field, you must manully upgrade to {product-title-short} 4.0.
+* You must be using {product-title-short} 3.74 to upgrade to {product-title-short} 4.0. If you are using a version older than 3.74, you must first upgrade to {product-title-short} 3.74 and then upgrade to {product-title-short} 4.0.
+====
+
+If you have installed {rh-rhacs-first} by using the {product-title-short} Operator, to upgrade to the latest version of {product-title-short} you must perform the following:
```

Comment:
You already use `{product-title-short}` in line 10

Reply:
Good catch, two writes were working on the same doc and I had to do some copy-pasta. Thank you.